### PR TITLE
Discord Team

### PR DESCRIPTION
**Pull request by Rajputusman**
Discord info mistake fix in newly updated merge

### CODE REVIEW

**Comment by Rajputusman** [comment]
Anywhere else I need to adjust Team information

File: src/types/teams.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ export enum TeamName {
   community = 'Community',
   design = 'Design',
   devOps = 'DevOps',
-  discordManagers = 'Discord Managers',
+  discordManagers = 'Discord',
```
Comment (thesanjeevsharma) [inline src/types/teams.ts:9]: can you also update the key to be `discord`?

**Review by thesanjeevsharma — CHANGES_REQUESTED** [review]
Also, update constants/teams.ts

Verify the changes by going to the teams page once.

File: src/types/teams.ts (hunk 1 above)
Comment (Rajputusman) [reply src/types/teams.ts]: Updated

**Review by thesanjeevsharma — APPROVED** [review]

**Pull request merged**